src/include/iscsi-lio-server/widgets.rb

```diff
@@ -643,6 +643,9 @@ def AddClntDialog
           if @changed_lun.has_key?(s)
             txt = _("Client name already exists!")
           end
+          if s.index /["\*\?"]/
```

Comment:
I have used 'index' like suggested in builtins.rb

   # Finds position of the first matching characters in string
    # @deprecated use {::String#index} instead
    def self.findfirstof string, chars
      return nil if string.nil? || chars.nil?

      return string.index /[#{Regexp.escape chars}]/
    end